versions/gefs.ver

```diff
@@ -1,8 +1,26 @@
-export gefs_ver=v11.2.0
-export gsm_ver=v12.1.3
-export post_ver=v5.1.2
-export enkf_ver=v2.0.3
-export grib_util_ver=v1.0.1
-export util_shared_ver=v1.0.5
-#export enkf_ver=`cat $NWROOTp1/versions/gfs.ver |grep enkf_ver |awk -F"=" '{print $2}'`
-#export post_ver=`cat $NWROOTp1/versions/gfs.ver |grep post_ver |awk -F"=" '{print $2}'`
+export gefs_ver=v12.0.0.0
```

Comment:
This is one too many version levels. GEFS should be v12.0.0

Reply:
Dingchen said "NCO will finalize the 3 section name. A fourth section give us more freedom before NCO finalization"

Reply:
Properly, semantic version numbers should be v12.0.0-0 or v12.0.0-beta if that is what he is going for https://semver.org/

Reply:
I will send an email to Dingchen and you to discuss the gefs_ver.